resources/serverless/charts/webhook/templates/cluster-role.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: {{ template "webhook.fullname" . }}
+rules:
```

Comment:
Please change it to: 

```yaml
- apiGroups: ["admissionregistration.k8s.io"]
  resources: ["mutatingwebhookconfigurations", "validatingwebhookconfigurations"]
  verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
- apiGroups: ["functions.serverless.kyma-project.io"]
  resources: ["functions", "functions/status"]
  verbs: ["get", "list", "create", "update", "patch", "watch"]
```

I don't know if `configmaps`, `secrets` etc are necessary. @aerfio

Reply:
Why? I think that the current format is more transparent

Reply:
Access to the `configmaps` and the `secrets` is necessary because of certificate and config stored in then and the webhook server read then in runtime.